requirements.txt

```diff
@@ -1,3 +1,2 @@
 Django==1.7.1
-wsgiref==0.1.2
```

Comment:
please do *not* remove this from the requirements file...
a.k.a. put it back :smile: